CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # sorted alphabetically
-* @gitbluf @jherrerasbp @sbp-bvanb
```

Comment:
Was this change intentional?

Reply:
I have added Marko without asking whether he wants to be codeowner. I will investigate the upcoming weeks and ask for consent who wants to join and do PR reviews.